corehq/apps/mocha/README.md

```diff
@@ -1,6 +1,33 @@
 Mocha Tests
 ===========
 
+## Getting started
```

Comment:
this is very similar to the instructions here: https://github.com/dimagi/commcare-hq#javascript-tests. what issues did you run into when following those? or do you just think they belong here?